com.tominlab.wonderpen.metainfo.xml

```diff
@@ -0,0 +1,66 @@
+    <p xml:lang="zh-cn">妙笔是一个强大且易用的写作工具，无论您是作家、学生还是写作爱好者，都可以使用它完成您的作品。</p>
+    <p>Features:</p>
+    <p xml:lang="zh-cn">功能特性：</p>
+    <ul>
+      <li>Tree-like multi-level directory, you can drag and drop to sort or change the nesting relationship at will, so that the directory structure of the long book is clear at a glance.</li>
+      <li xml:lang="zh-cn">树状多级目录，可随意拖拽排序或改变嵌套关系，让长篇巨著的目录结构一目了然。</li>
+      <li>An easy-to-use text editor that supports Markdown syntax.</li>
+      <li xml:lang="zh-cn">简单易用的文本编辑器，支持 Markdown 语法。</li>
+      <li>It can be switched to a full-screen composition mode to eliminate distractions and concentrate on writing.</li>
+      <li xml:lang="zh-cn">可切换为全屏专注模式，排除干扰，专心写作。</li>
+      <li>Easily export to image, PDF, Epub, Word, HTML and other formats.</li>
+      <li xml:lang="zh-cn">便捷地导出为图片、PDF、Epub、Word、HTML 等多种格式。</li>
+      <li>Supports whiteboard mode, where you can add notes, images, checklist, tables, kanban and other content arbitrarily.</li>
+      <li xml:lang="zh-cn">支持白板模式，可在其中任意添加便笺、表格、看板等内容。</li>
+      <li>Individual memos can be added to each document.</li>
+      <li xml:lang="zh-cn">每个文档都可以添加单独的备注。</li>
+      <li>At the same time, it supports global note cards, which is convenient for reference when writing across documents.</li>
+      <li xml:lang="zh-cn">同时支持全局备注卡片，方便跨文档写作时参考。</li>
+      <li>When there are too many documents, it can focus on a node in the document tree and hide others.</li>
+      <li xml:lang="zh-cn">当文档太多时，可聚焦于文档树中的一个节点，隐匿其余细节。</li>
+      <li>Support full-text search and replace.</li>
+      <li xml:lang="zh-cn">支持全文搜索及替换。</li>
+      <li>Support chapter splitting and merging.</li>
+      <li xml:lang="zh-cn">支持章节切分以及合并。</li>
+      <li>Documents are automatically saved after modification.</li>
+      <li xml:lang="zh-cn">文档修改后自动保存。</li>
+      <li>Support document snapshot and full library backup, support automatic backup, to ensure data security without losing manuscripts.</li>
+      <li xml:lang="zh-cn">支持文档快照以及全库备份，支持自动备份，确保数据安全不丢稿。</li>
+      <li>Supports dark mode, and has built-in multiple themes such as green, yellow, and gray.</li>
+      <li xml:lang="zh-cn">支持夜间模式，还内置了浅绿、淡黄、深灰等多个主题。</li>
+      <li>Provides a large number of personalized settings, and can also add custom CSS to meet various usage habits.</li>
+      <li xml:lang="zh-cn">提供大量个性化设置，还可自定义 CSS，满足多种使用习惯。</li>
+    </ul>
+    <p>Hope you enjoy this app as much as we do. If you have any questions or suggestions during use, you can contact us through the "Feedback" option in the "Help" menu, or send us an email: support@tominlab.com .</p>
+    <p xml:lang="zh-cn">希望您像我们一样，喜爱这个应用。使用过程中有任何问题或建议，都可以通过「帮助」菜单中的「意见反馈」选项联系我们，或者也可以给我们发送邮件：support@tominlab.com。</p>
+  </description>
+  <screenshots>
+    <screenshot type="default">
+      <caption>Writing</caption>
+      <image type="source">https://cdn.tominlab.com/upload/2023/08/12/wonderpen-en.png/image>
```

Comment:
You can add the CN screenshot too with the xml:lang tag see, https://docs.flathub.org/docs/for-app-authors/metainfo-guidelines/#metainfo-translations